dcos/http.py

```diff
@@ -91,10 +64,12 @@ def request(method,
     try:
         with requests.Session() as session:
             response = session.send(request.prepare(), timeout=timeout)
-    except Exception as ex:
-        logger.exception('Error making HTTP request: %r', request.url)
-
-        raise to_exception(ex)
+    except requests.exceptions.ConnectionError as e:
```

Comment:
Why do this if we're effectively re-raising the exception?  We're not losing any info.